### PR TITLE
LPD-29488 [DataSetViewSorting.testcase] Delete poshi

### PR DESCRIPTION
**Pull request by thektan**
https://liferay.atlassian.net/browse/LPD-32245
https://liferay.atlassian.net/browse/LPD-29488

---

- Removes the `DataSetViewSorting.testcase` poshi test and adds any missing tests into Playwright.
- Also includes [LPD-32245](https://liferay.atlassian.net/browse/LPD-32245) to fix the incorrect language key used

### CODE REVIEW

**Comment by thektan** [comment]
ci:test:playwright

**Comment by thektan** [comment]
ci:test:relevant

File: modules/apps/frontend-data-set/frontend-data-set-admin-web/src/main/resources/META-INF/resources/js/data_set/sorting/Sorting.tsx [hunk 1]
```diff
@@ -579,7 +579,7 @@ const Sorting = ({
 				},
 			],
 			status: 'warning',
-			title: Liferay.Language.get('delete-filter'),
+			title: Liferay.Language.get('delete-sorting'),
```
Comment (dsanz) [inline modules/apps/frontend-data-set/frontend-data-set-admin-web/src/main/resources/META-INF/resources/js/data_set/sorting/Sorting.tsx:582]: good catch!

**Comment by thektan** [comment]
ci:forward

**Pull request closed without merging**